config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get "/:community_id/memberships", to: "memberships#new"
+  post "/:community_id/memberships", to: "memberships#create"
```

Comment:
It feels odd to me that we can't use nested resources or something like that?

Reply:
I tried using nested resource and shallow to hide the `/communities/` since I think it might be important to keep the path simple for people to join (assuming some people might just type the path in).

Is it a hard requirement to do `/:community_id/memberships` or can we do `communities/:community_id/memberships`?

I can revisit that after styling the view.

Reply:
I guess the way I am thinking of this is that when you go to nourish.is you get presented with each of the communities so you can browse them.

Then, when you click on one it goes to `/organizing/community_slug` which has the communities sign up form.

Signing up for a community, if you already are logged in, will create a community membership record in the pending state that joins the person and the community.

If you aren't logged in, it will create a person record in the database, as well as a community membership record.